dbcon/joblist/joblist.cpp

```diff
@@ -672,7 +672,7 @@ void JobList::graph(uint32_t sessionID)
     else if (typeid(*(qsi->get())) == typeid(CrossEngineStep))
     {
       tcn.schema = qsi->get()->schema();
-      tcn.table = qsi->get()->alias();
+      tcn.table = qsi->get()->alias(); // XXX is it correct? why not table name?
```

Comment:
Alias are more specific than tables. People tend to know what do  their aliases mean in the context of a query, even in case of CEJ.